### PR TITLE
Initial Film models

### PR DESCRIPTION
**Pull request by jcjudkins**

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[[source]]
```
Comment (copelco) [inline Pipfile]: Remove this file

File: example-facets/films/models.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+#     (5, "TV-14"),
+#     (6, "TV-Y"),
+#     (7, "TV-Y7"),
+#     (8, "G"),
+#     (9, "TV-G"),
+#     (10, "PG"),
+#     (11, "TV-PG"),
+# )
+# Create your models here.
+class films(models.Model):
```
Comment (copelco) [inline example-facets/films/models.py]:
```suggestion
class Film(models.Model):
```

File: example-facets/films/models.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+#     title = models.CharField(max_length=100)
+#     director = models.CharField(max_length=100)
+#     cast = models.TextField()
+#     country = models.CharField(max_length=25)
+#     date_added = models.DateField()
+#     release_year = models.DateField()
+#     rating = models.PositiveSmallIntegerField(choices=RATING)
+#     duration = models.DurationField()
+#     listed_in = models.TextField()
+#     description = models.TextField()
```
Comment (copelco) [inline example-facets/films/models.py]:
```suggestion
```

**Review by copelco — COMMENTED** [review]
Added some initial comments. Also remove the `.DS_Store` and `.zip` files please.

File: example-facets/films/models.py (hunk 2 above)
Comment (jcjudkins) [reply example-facets/films/models.py]: Made suggested change.

File: example-facets/films/models.py (hunk 3 above)
Comment (jcjudkins) [reply example-facets/films/models.py]: Made suggested change.

File: Pipfile (hunk 1 above)
Comment (jcjudkins) [reply Pipfile]: Removed Pipfile

**Review by copelco — APPROVED** [review]

**Pull request merged**